packages/itwinui-css/backstop/tests/alert.html

```diff
@@ -14,15 +14,12 @@
       content="width=device-width, initial-scale=1"
     />
     <title>Alert | iTwinUI</title>
-    <link
-      rel="stylesheet"
-      href="../../css/all.css"
-    />
-    <link
-      rel="stylesheet"
-      href="../assets/demo.css"
-    />
+    <script type="module">
+      import "./assets/theme.js";
+    </script>
     <style>
+      @import url("./assets/demo.css") layer(demo);
+      @import url("@itwin/itwinui-css/css/all.css") layer(itwinui);
```

Comment:
I'm using CSS layers here to work around this Vite bug which messes up order of CSS (thus causing visuals to change): https://github.com/vitejs/vite/issues/3924

VSCode will currently show red squigglies even though this is valid syntax. Should be fixed soon: https://github.com/microsoft/vscode-css-languageservice/pull/276

Reply:
Don't you need to set layer order?

Reply:
You say if we swapped demo and all css files order, it got messed up??? Should have fixed demo.css instead.

Reply:
Also, it shows this as error
![image](https://user-images.githubusercontent.com/81580355/180172526-ce54b4ab-e45c-492f-a389-adf0ecc00ca1.png)

Reply:
> You say if we swapped demo and all css files order, it got messed up??? Should have fixed demo.css instead.

No it's about the bespoke styles present in the `.html` pages (inside `<head>`). Vite places imported styles at the end of `<head>`, which overrides those page styles. The only other way to get around it would be to increase specificity in every .html page.

> Also, it shows this as error

see above: https://github.com/iTwin/iTwinUI/pull/708#discussion_r922521350